docs/source/widgets.rst

```diff
@@ -0,0 +1,137 @@
+.. _widgets:
+
+=======
+Widgets
+=======
+
+.. py:currentmodule:: pyface
+
+A Widget in Pyface is an object wraps a control from the underlying toolkit
+and provides standard traits and methods that abstract the differences
+between the interfaces presented by the toolkit widgets.
+
+The public API for every widget is provided by the appropriate interface
+class: :class:`~pyface.i_widget.IWidget` and its subclasses such as
+:class:`~pyface.i_application_window.IApplicationWindow` and
+:class:`~pyface.fields.i_text_field.ITextField`.
+
+The actual implementation is provided by the corresponding toolkit-dependent
+:class:`~pyface.widget.Widget` subclass, but with shared functionality split
+into separate mixins: :class:`~pyface.i_widget.MWidget` and so on, following
+the general design described in the :ref:`overview` section.
+
+Lifecycle
+=========
+
+When a :class:`~pyface.widget.Widget` instance is created only the traits
```

Comment:
After running `python etstool.py docs --toolkit=pyqt5`, the classes are merely bolded but they are not links to anything.
![Screenshot 2020-10-13 at 16 39 46](https://user-images.githubusercontent.com/3673984/95883440-d5d6f780-0d72-11eb-8f89-0de0cc25a70f.png)

Just to confirm, I can import `pyface.widget.Widget` from the same environment I used for building the doc.

Reply:
I'm guessing that the autodoc tools are not seeing the classes being imported there because of toolkit magic.

Fine for now; but should open an issue because there may be other places thiis is happening.